analyzer/src/main/kotlin/curation/Sw360PackageCurationProvider.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2020-2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.curation
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.eclipse.sw360.antenna.http.HttpClientFactoryImpl
```

Comment:
Empty line above.

Reply:
BTW, this should now be checked automatically thanks to https://github.com/oss-review-toolkit/ort/pull/3494.